### PR TITLE
Added array example to getStaticPaths()

### PR DESCRIPTION
**Pull request by lostra01**
Added a quick array example to getStaticPaths() for those who want to generate data based on a set of locally available ones, but do not use fetch or other functions to retrieve them.

### CODE REVIEW

**Review by aFuzzyBear — APPROVED** [review]
lgtm

File: src/pages/en/reference/api-reference.md [hunk 1]
```diff
@@ -166,6 +166,30 @@ const {post} = Astro.props;
 <body><h1>{id}: {post.name}</h1></body>
 ```
 
+Alongside of using `await fetch()` you can also pass manually constructed or generated arrays to a `map` function, and let Astro generate all the pages for you, based on the data you have provided.
```
Comment (jonathantneal) [inline src/pages/en/reference/api-reference.md:169]: I like this example. May we shorten this phrase?

```suggestion
You can also pass a regular array, which may be helpful when generating or stubbing a known list of routes.
```

**Review by jonathantneal — CHANGES_REQUESTED** [review]
I would like to add this. Thank you, @lostra01!

I left a small editorial comment. Would you please take a look?

File: src/pages/en/reference/api-reference.md (hunk 1 above)
Comment (jonathantneal) [reply src/pages/en/reference/api-reference.md:169]: From discussion:
> Thanks for the edit! Actually makes it more readable! 🙂

Committing the suggestion and merging :)

**Comment by jonathantneal** [comment]
Merged, @lostra01. 🎉

**Pull request merged**